scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -1090,75 +1095,30 @@ private void setAllValuesAsString() {
         };
     }
 
-    private com.google.gwt.event.dom.client.ClickHandler clickHandlerForUploadFromFileButton(final FormPanel toSubmit) {
-        return new com.google.gwt.event.dom.client.ClickHandler() {
-            @Override
-            public void onClick(com.google.gwt.event.dom.client.ClickEvent event) {
-                String fileName = fileUpload.getFilename();
-                if ("".compareTo(fileName) != 0) {
-                    displayLoadingMessage();
-                    toSubmit.submit();
-                } else {
-                    displayErrorMessage("Nothing to upload. Please select a file.");
-                }
-
+    private com.google.gwt.event.dom.client.ClickHandler clickHandlerForUploadFromFileButton(FormPanel toSubmit) {
```

Comment:
Can we remove the package mention here?

Reply:
Done thx

Reply:
@jrochas There is another ClickHandler from another package this is widely used in here so I should remain as is to avoid conflicts